docs/local-deployment/overview.mdx

```diff
@@ -8,16 +8,16 @@ description: Deploy Self-hosted Aserto locally
 
 Aserto authorizers are built to be deployed right next to your application, in your cloud.
 
-## Aserto Sidecar
+## Edge Authorizer
 
-The simplest local deployment is in the form of a single container, known as the **Aserto Sidecar**. You can deploy it either as a local microservice (and scale it horizontally), or if you're running in Kubernetes, you can deploy it as a sidecar in your application pod.
+The simplest local deployment is in the form of a single Topaz container that is configured as an Aserto Edge Authorizer. You can deploy it either as a local microservice (and scale it horizontally), or if you're running in Kubernetes, you can deploy it as a sidecar in your application pod.
 
-The Sidecar container implements the Aserto Directory using an embedded database, based on BoltDB. It scales up to a couple of GB of data (which could house 10k's of users, depending on how many properties each user contains).
+When running the Edge Authorizer it contains an embedded Aserto Directory which uses an embedded database, based on BoltDB. It scales up to a couple of GB of data (which could house 10k's of users, depending on how many properties each user contains).
```

Comment:
wdyt about "The Edge Authorizer implements the Aserto Directory (link) using an embedded database that is based on BoltDB. The database scales...."

Just a thought. the double "embedded" in the sentence was the main thing I was trying to fix

Reply:
Yeah I didn't like Embedded twice. How about "includes an Aserto Directory that is implemented using...."